pkg/controller/controller_instance.go

```diff
@@ -1539,9 +1608,19 @@ func (c *controller) processProvisionFailure(instance *v1beta1.ServiceInstance,
 	// requeue this resource.
 	var err error
 	if shouldMitigateOrphan {
+		// Copy failure reason/message to a new OrphanMitigation condition
```

Comment:
@staebler please check below if that's what you had in mind for preserving reason/message in OrphanMitigation condition (as well as overwriting the Ready condition)

Reply:
@nilebox Yep, that is what I had in mind.

As an aside, I am not sure why we are creating `orphanMitigationCond` and `readyCond` just to read the status, reason, and message from the conditions.